bitnami/mongodb/Chart.yaml

```
@@ -39,4 +39,4 @@ maintainers:
 name: mongodb
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/mongodb
-version: 14.7.3
+version: 14.7.4
```

We should bump the minor version here as, per semver standards, we are adding a new feature

```suggestion
version: 14.8.0
```